osu.Game/Overlays/Social/Header.cs

```diff
@@ -53,12 +54,12 @@ private void load(OsuColour colours)
 
     public enum SocialTab
     {
-        [Description("Online Players")]
-        OnlinePlayers,
-        //[Description("Online Friends")]
-        //OnlineFriends,
-        //[Description("Online Team Members")]
-        //OnlineTeamMembers,
+        [Description("All Players")]
+        AllPlayers = SocialSortCriteria.Rank,

```

Comment:
not sure about these `= ...` pieces here. should probably not be a thing.

Reply:
Probably not needed anymore. Agreed.